gpii/node_modules/lifecycleManager/src/LifecycleManagerSession.js

```diff
@@ -59,10 +59,9 @@ var gpii = fluid.registerNamespace("gpii");
         members: {
             createTime: "@expand:Date.now()",
             // TODO: Refactor this into some more satisfactory future pattern - some kind of lightweight version of "transforming promise chain"
```

Comment:
Shall this "TODO" line be removed with the change at line 62 that combining more than one fetcher is no longer necessary?